src/main/java/nl/tudelft/semgroup4/util/PlayerScorePair.java

```diff
@@ -0,0 +1,20 @@
+package nl.tudelft.semgroup4.util;
+
+public class PlayerScorePair {
```

Comment:
I would rename this to something like highscoreEntry. Now it is unclear what the pair is used for.